src/order_book/response.rs

```diff
@@ -0,0 +1,92 @@
+use derive_getters::Getters;
+use serde::{Deserialize, Serialize};
+
+use crate::models::Response;
+
+/// Represents the response from the Horizon server when querying for the details of an order book.
+/// 
+/// This struct represents the response from the Horizon server when querying for the details of an order book.
+/// It includes the bids, asks, base, and counter fields.
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize, Getters)]
+#[serde(rename_all = "camelCase")]
+pub struct DetailsResponse {
+    /// The bids in the order book.
+    pub bids: Vec<Bid>,
+    /// The asks in the order book.
+    pub asks: Vec<Ask>,
+    /// The base asset of the order book.
+    pub base: Base,
+    /// The counter asset of the order book.
+    pub counter: Counter,
+}
+
+/// Represents a bid in the order book.
+/// 
+/// This struct represents a bid in the order book. It includes the price_r, price, and amount fields.
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize, Getters)]
+#[serde(rename_all = "camelCase")]
+pub struct Bid {
+    #[serde(rename = "price_r")]
+    pub price_r: PriceR,
+    pub price: String,
+    pub amount: String,
+}
+
+/// Represents the price_r field in the order book.
+/// 
+/// This struct represents the price_r field in the order book. It includes the n and d fields.
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize, Getters)]
+#[serde(rename_all = "camelCase")]
+pub struct PriceR {
+    pub n: i64,
+    pub d: i64,
```

Comment:
Kunnen we `n` en `d` een logische naam om documentatie geven? Ik heb zo 123 geen idee wat deze waardes inhouden.